src/Model/Resolver/CartDisplayConfigResolver.php

```diff
@@ -0,0 +1,140 @@
+<?php
```

Comment:
This file looks copied, can you please add information which you replaced there / which was changed? 

Otherwise, it is OK

Reply:
This file is not copied.
Just tried to make it look like other code.